docs/sources/administration/data-source-management/index.md

```diff
@@ -81,51 +82,47 @@ You can assign data source permissions to users, teams, and roles which will all
+
+A panel's interval is visible in the [query options]({{< relref "../../panels-visualizations/query-transform-data/" >}}). It is calculated like this: `time range / max data points`. Max data points are calculated based on the width of the panel. For example, a wide panel with `1000 data points` on a dashboard with a time range of `last 7 days` will retrieve fresh data every 10 minutes: `7d / 1000 = 10m`. In this example, cached data for this panel will be served for up to 10 minutes before Grafana needs to query the data source again for new data.
 
-Interval is visible in a panel's [query options]({{< relref "../../panels-visualizations/query-transform-data/" >}}). It is calculated like this: `(max data points) / time range`. Max data points are calculated based on the width of the panel. For example, a full-width panel on a dashboard with a time range of `last 7 days` will retrieve fresh data every 10 minutes. In this example, cached data for this panel will be served for up to 10 minutes before Grafana queries the data source again and returns new data.
+You can make a panel retrieve fresh data more often by increasing the **Max data points** setting in the panel's [query options]({{< relref "../../panels-visualizations/query-transform-data/" >}}).
 
-You can make a panel retrieve fresh data more frequently by increasing the **Max data points** setting in the panel's [query options]({{< relref "../../panels-visualizations/query-transform-data/" >}}).
+### Caching benefits
 
-### Query caching benefits
+By reducing the number of queries and requests sent to data sources, caching can provide the following benefits:
 
 - Faster dashboard load times, especially for popular dashboards.
 - Reduced API costs.
 - Reduced likelihood that APIs will rate-limit or throttle requests.
 
 ### Data sources that work with query caching
 
-Query caching works for all [Enterprise data sources](/grafana/plugins/?type=datasource&enterprise=1) as well as the following [built-in data sources]({{< relref "../../datasources/" >}}):
-
-- CloudWatch Metrics
-- Google Cloud Monitoring
-- InfluxDB
-- Microsoft SQL Server
-- MySQL
-- Postgres
-- Tempo
+Query caching works for all backend data sources. More specifically, caching works with Grafana's [built-in data sources]({{< relref "../../datasources/#built-in-core-data-sources" >}}), and any [backend data source plugins](https://grafana.com/grafana/plugins/?type=datasource) that extend the `DataSourceWithBackend` class in the plugins SDK.
 
-Some data sources, such as Elasticsearch, Prometheus, and Loki, cache queries themselves, so Grafana query caching does not improve performance.
+To verify that a data source works with query caching, follow the instructions below to **Enable and Configure query caching**. If caching is enabled in Grafana but the Caching tab is not visible for the given data source, then query caching is not available for that data source.
```

Comment:
Add a link to the instructions below.

Reply:
Done on [3b46638](https://github.com/grafana/grafana/pull/70556/commits/3b46638c9afcdda4905bb4941b9d069440705565)